### PR TITLE
Update quotes.txt with more quotes

### PR DESCRIPTION
**Pull request by Dazpoet**
No code additions, humour not guaranteed, I hope @plague006 is ok with the jab but to me it was a very distinct thing which we all should remember :D

### CODE REVIEW

**Comment by plague006** [comment]
So pained was he by seeing homeless mods that he actually moved out of his comfortable metadata and slammed his head against C# until https://github.com/KSP-CKAN/CKAN/pull/1254 was born.

**Comment by ChucklesTheBeard** [comment]
> \* pjf (~pjf@216.46.28.66) has joined
\<ChucklesTheBeard> ayy
\<p6-away> hey boss
\<pjf> Yes. Chuckles is boss now. This pleases me. :)

**Comment by plague006** [comment]
>[04:30] (@plague006): politas, Olympic1 you guys around?
[04:30] (@plague006): y'all [goofed] :p
[04:32] *** Olympic1 is now known as Olympic1|AWAY

**Comment by Dazpoet** [comment]
Gonna do the ungodly awful thing here and merge my own PR. Pretty sure it won't break anything :p

File: quotes.txt [hunk 1]
```diff
@@ -14,3 +14,39 @@ Windows is "I'm locking my sandwich in this safe. Why are you
...
+
+    -- plague006 all early PRs
+%
+<ChucklesTheBeard> ayy
+<p6-away> hey boss
+<pjf> Yes. Chuckles is boss now. This pleases me. :)
+
+    -- pjf on irc
+%
+Sometimes you just don't want to fix things.
```
Comment (Olympic1) [inline quotes.txt]: That was my plugin that sets me in away status when my computer shuts down or sleepmode.

File: quotes.txt [hunk 2]
```diff
@@ -14,3 +14,39 @@ Windows is "I'm locking my sandwich in this safe. Why are you
 worried it will go stale?"
 
     --  pjf, IRC
+%
+Jenkins is refusing to work a simple fix and keeps reporting the
+build as broken. 3 different people are trying to figure out why
+NetKAN reports an error and calls a forth one to come check it out.
+
+"licence" is misspelled
+
```
Comment (techman83) [inline quotes.txt]: :laughing: :laughing: we should write an extra test for typos :+1:

Comment (politas) [inline quotes.txt]: Strictly speaking, that wasn't a typo. It was an alternate spelling, perfectly correct linguistically, but wrong programatically

**Pull request merged**